install.py

```diff
@@ -43,7 +43,8 @@
 
 
 NSC_REPO_URL = "https://github.com/nats-io/nsc"
-NSC_LATEST_RELEASE_URL = NSC_REPO_URL + "/releases/latest"
+# NSC_LATEST_RELEASE_URL = NSC_REPO_URL + "/releases/latest"
+NSC_PROD_RELEASE_URL = NSC_REPO_URL + "/releases/tag/0.5.0"
```

Comment:
think it could be v1.0.0 as well

Reply:
ah no it has to be v0.5.0 because there are no artifacts for the v1.0.0 tag :/ 
https://github.com/nats-io/nsc/releases/tag/1.0.0